packages/components/src/box-control/README.md

```diff
@@ -48,10 +48,11 @@ If this property is true, when the box control is unlinked, vertical and horizon
 
 ### inputProps
 
-Props for the internal [InputControl](../input-control) components.
+Props for the internal [UnitControl](../unit-control) components.
 
 -   Type: `Object`
 -   Required: No
+-   Default: `{ min: 0 }`
```

Comment:
These props are actually passed through to UnitControl, not InputControl. Why does it matter? `InputControlProps[ 'max' ]` is `string` but `UnitControlProps[ 'max' ]` is `number` ☠️

Reply:
😵‍💫  Hopefully this will be progressively addressed as we work our way up from `NumberControl`